Jenkinsfile

```diff
@@ -26,7 +34,15 @@ elifeLibrary {
             elifeGitMoveToBranch commit, 'master'
         }
 
-        stage 'Push image', {
+        stage 'Push release image', {
+            isNew = sh(script: "git tag | grep v${candidateVersion}", returnStatus: true) != 0
+            if (isNew) {
+                def image = DockerImage.elifesciences(this, 'sciencebeam', commit)
+                image.tag(candidateVersion).push()
```

Comment:
should also push the latest release here

Reply:
also these two could be swapped, first push `latest` and after that the release; if anything goes wrong you may have a `latest` without a corresponding release (allowed) but never a release that diverges from `latest`

Reply:
(same as "always push `master` before tags from `master`"

Reply:
We had that order before (push latest after commit). Having a latest without the release means something went wrong and maybe we shouldn't use it. But will comply.

Reply:
We should discuss some standard across projects, in general I tend to follow `commit` -> `master`|`latest` -> `tag`|release

Reply:
I agree in that case we shouldn't use it, because we would have pinned on the previous release tag